AdaSupport/embed
# FND-1119: Open/Close Tabbing Navigation Fix

**What**
Short fix to an issue where the IntroBlurb and its corresponding close button could still be tabbed to when it was closed. 

**How**
Previously, a `tabIndex={0}` attribute was added to the \<p\> tag of the IntroBlurb so that it could be tabbed to as a link. The corresponding close \<button\> could be tabbed to by default. However, this attribute wasn't responsive to the IntroBlurb state, and it could be tabbed to when the blurb was closed. 

This fix adds a state-responsive `tabIndex={animateIntroOut ? -1 : 0}` attribute to the IntroBlurb \<p\> and close \<button\> tags.

**How to Test**
Create an intro in the embed testing page, and press tab when the intro appears. You should be able to tab through the intro's close button and the intro itself. When it disappears, you should be able to tab directly from the previous page element to the question mark button, without tabbing the invisible IntroBlurb component. 

---
